tuna/connection.py

```diff
@@ -133,14 +133,17 @@ def is_alive(self):
       return not self.out_channel.exit_status_ready()
 
   def exec_command_unparsed(self, cmd, timeout=SSH_TIMEOUT, abort=None):
-    """Function to exec commands"""
+    """Function to exec commands
+
+    warning: leaky! client code responsible for closing the resources!
```

Comment:
what is this leaky !client code about?

Reply:
Opens opens a process to run a command in, but doesn't close it. So whoever uses the function is responsible for closing the process

Reply:
I'll add this to the comment